### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by karanJ2212**
Set up React Testing Library.

Written unit tests for the files operate.js and calculate.js using Jest.
Created unit tests for all React components:
Used Jest snapshots to test the components.
Used React Testing Library to simulate user interaction.
All tests should passed.

### CODE REVIEW

File: src/tests/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  });
+  test('Return the difference of two numbers', () => {
+    expect(operate(4, 1, '-')).toEqual('3');
+  });
+  test('Multiply two numbers', () => {
+    expect(operate(62, 10, 'x')).toEqual('620');
+  });
+  test('Divide two numbers', () => {
+    expect(operate(9, 3, '÷')).toEqual('3');
+  });
```
Comment (Munsa1) [inline src/tests/operate.test.js:15]: - [ ] Kindly add a test for the modulus `%` operation as this is a requirement

File: src/tests/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    expect(calculate(Obj, '=')).toStrictEqual({ total: '15', next: null, operation: null });
+  });
+
+  test('Passing objects of two numbers and - operator to get their difference', () => {
+    Obj = calculate(Obj, '100');
+    Obj = calculate(Obj, '-');
+    Obj = calculate(Obj, '1');
+    expect(calculate(Obj, '=')).toStrictEqual({ total: '99', next: null, operation: null });
+  });
+
```
Comment (Munsa1) [inline src/tests/calculate.test.js:39]: - [ ] Again here kindly add a test case senario for the `&` modulus operation as this is a requirement

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @karanJ2212  @Patfarmurs , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @karanJ2212 @Patfarmurs , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linters checks :heavy_check_mark: 
gitflow :heavy_check_mark: 
pair-programming :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**